packages/nimble-components/src/banner/template.ts

```diff
@@ -0,0 +1,42 @@
+import { html, slotted, when } from '@microsoft/fast-element';
+import { DesignSystem } from '@microsoft/fast-foundation';
+import type { Banner } from '.';
+import { Button } from '../button';
+import { IconExclamationMark } from '../icons/exclamation-mark';
+import { IconInfo } from '../icons/info';
+import { IconTriangleFilled } from '../icons/triangle-filled';
+import { IconXmark } from '../icons/xmark';
+
+// prettier-ignore
+export const template = html<Banner>`
+    <template role="status" aria-label=${x => x.getAriaLabel()}>
+        <div class="icon">
+            ${when(x => x.severity === 'error', html`
+                <${DesignSystem.tagFor(IconExclamationMark)}></${DesignSystem.tagFor(IconExclamationMark)}>
+            `)}
+            ${when(x => x.severity === 'warning', html`
+                <${DesignSystem.tagFor(IconTriangleFilled)}></${DesignSystem.tagFor(IconTriangleFilled)}>
+            `)}
+            ${when(x => x.severity === 'info', html`
+                <${DesignSystem.tagFor(IconInfo)}></${DesignSystem.tagFor(IconInfo)}>
+            `)}
+        </div>
+        <div class="text">
+            ${when(x => !x.titleHidden, html`
+                <slot name="title" ${slotted('slottedTitle')}></slot>
+            `)}
+            <slot></slot>
+        </div>
+        <div class="controls">
+            <slot name="action"></slot>
+            <div class="dismiss">
+                ${when(x => !x.preventDismiss, html<Banner>`
```

Comment:
You can tab to it, but it wasn't getting styled any differently. I think that is because I am overriding the border hover color with `transparent`. I ended up giving it an outline so there is some visual indication. We don't have an explicit design from Brandon for this, and I think this button is too small to do the usual double outline.